BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -57,6 +57,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('transformer.yml');
         $loader->load('icon.yml');
         $loader->load('leftpanel.yml');
+        $loader->load('value_transformer.yml');
```

Comment:
in the yaml file name, you should not put any underscore. This file should be named : `valuetransformer.yml`